spec/index.html

```diff
@@ -1362,6 +1367,16 @@ <h2>Internationalization Considerations</h2>
   <p class="issue" data-number="9"></p>
 </section>
 
+<section id="iri-abnf" class="appendix informative">
+  <h2>IRI Grammar</h2>
+
+  <p>The following [[ABNF]] grammar is from [[RFC3987]] and includes the referenced
+    productions from [[RFC3986]].</p>
+
+  <div data-include="iri-grammar.html"></div>
```

Comment:
```suggestion
  <div data-include="iri-grammar.abnf" data-include-format='text'></div>
```
This may need to be include markdown and put `\`` ... \``` around the ABNF inclusion.

Ideally, the inclusion would be just the text and there would be a URL text/plain that can be read directly.

Reply:
I'll work on this.